### PR TITLE
Support dbt utils breaking changes

### PR DESCRIPTION
**Pull request by IDoneShaveIt**
dbt are about to make a breaking change in dbt-utils (deprecating macros and moving them into dbt-core).
In order to use those macros without making our users upgrading their dbt-core/utils, we wrapped those `cross-db-utils` to support all versions.

### CODE REVIEW

File: macros/utils/cross_db_utils/type_bigint.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{% macro type_bigint() %}
```
Comment (Maayan-s) [inline macros/utils/cross_db_utils/type_bigint.sql:1]: We have a data_types folder on our utils, maybe it's a better fit (all the type macros)

File: macros/utils/cross_db_utils/timediff.sql [hunk 2]
```diff
@@ -1,3 +1,4 @@
+{# Same as datediff, but used on timestamps instead #}
```
Comment (Maayan-s) [inline macros/utils/cross_db_utils/timediff.sql:1]: This works for both timestamp and date (fix the comment, it's unclear)

File: macros/utils/cross_db_utils/timeadd.sql [hunk 3]
```diff
@@ -1,3 +1,4 @@
+{# Same as dateadd, but used on timestamps instead #}
```
Comment (Maayan-s) [inline macros/utils/cross_db_utils/timeadd.sql:1]: This works for both timestamp and date (fix the comment, it's unclear)

File: macros/utils/cross_db_utils/current_timestamp.sql [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{% macro current_timestamp() -%}
```
Comment (Maayan-s) [inline macros/utils/cross_db_utils/current_timestamp.sql:1]: Is this just copying from dbt utils? 
If so we need to credit them here and add the licensing

File: macros/utils/cross_db_utils/type_bigint.sql (hunk 1 above)
Comment (IDoneShaveIt) [reply macros/utils/cross_db_utils/type_bigint.sql:1]: I just wanted to make it clear that this is created to support the deprecation of dbt-utils.
But sure I can move it over there..

File: macros/utils/cross_db_utils/current_timestamp.sql [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License. 
+#}
+
+{# 
+    dbt_utils.current_timestamp is using dbt_utils.type_timestamp which is going to be deprecated from dbt-utils version 0.9.0,
+    therefore dbt_utils.current_timestap is going to be deprecated as well (without having a similar macro at dbt-core).
+    We copied this macro, with a small change that supports the deprecation, so we could continue and use it in our project.
+#}
```
Comment (Maayan-s) [inline macros/utils/cross_db_utils/current_timestamp.sql:21]: @oravi do you think this is ok? we give them the copyright and explain why we had to duplicate.

**Comment by Maayan-s** [comment]
LGTM, just want @oravi to say that he is ok with the license and comment

**Pull request merged**